scoping_document.md

```diff
@@ -112,48 +112,48 @@ The corresponding acceptance criteria supplement the specification of the requir
+| E01.04 | As an app user, the first time I use the app, I want to be asked whether the app can access the smartphone’s Bluetooth function, so that I can control how the app is used on the smartphone. | 1. This user story is tantamount to E01.03. The authorization to use the generation of pseudonymous IDs and send them to nearby devices is already contained in the usage of Bluetooth Low Energy (BLE).<hr/>2. The configuration of the general Bluetooth functions is only possible in the system settings. |
+| E01.05 | As an app user, the first time I use the app, I want to be asked whether the app is allowed to send me notifications, so that the app can provide me with relevant information without requiring me to open the app proactively. | 1. The app’s notification settings are queried before the app is used for the first time. This enables local notifications. True push notifications from external servers are not supported (APNs/FCM). <hr/>2. The prompt no longer appears after the first time the app is used. |
+| E01.06 | As an app user, I want the app to be displayed in my language the first time I use the app, so that I can understand how to use the app. | 1. The configured system language is detected.<hr/>2. If the content is not available in the detected system language, English is selected by default.<hr/>3. If the configured system language is German, the app language is set to German. |
+| E01.07 | As an app user, I want to be able to access help functions and settings for accessibility during the onboarding process, so that I can use the app despite technical difficulties or disabilities. | 1. Accessibility is provided depending on the options available in the respective operating system. |
 
 ### Information and Instructions for Using the App
 | # of user story ID | User story | Acceptance criteria |
 |-----------------|------------|--------------------|
-| E02.01 | As an app user, I want to have access to a FAQ list, so that I can find answers to questions I might have about the app. | 1. The app will either contain a link to a web page with FAQs, which is displayed in a browser window, or the web page will be displayed within the app itself. |
-| E02.02 | As an app user, I want to have access to instructions, so that I can understand the app and its functions. | 1. An explanation of the app’s various functions will be provided. |
+| E02.01 | As an app user, I want to be able to read an FAQ list, so that I can find answers to questions I have about the app. | 1. The app will either contain a link to a web page with FAQs, which is displayed in a browser window, or the web page will be displayed within the app itself. |
+| E02.02 | As an app user, I want to have access to instructions, so that I can understand the app and the app's functions. | 1. An explanation of the app’s various functions will be provided. |
```

Comment:
> so that I can understand the app and **the app's** functions.

I don't agree with this change. I find "its" to be more fitting

Reply:
I guess this is a matter of taste. Pronouns are frowned upon in requirements engineering (cf.  ISO29148:2011, p.12) but I guess there is no real risk of ambiguity or readability here. I guess both are fine and would be open to revert depending on other people's opinions.

Reply:
> Pronouns are frowned upon in requirements engineering (cf. ISO29148:2011, p.12)

Interesting! Never heard of that. Thank you

Reply:
The standard discourages "vague and general terms", because they would "result in requirements that are often difficult or even impossible to verify or may allow for multiple interpretations" Also the standard lists some typical terms to avoid, among them: "Vague pronouns (such as 'it', 'this', 'that')". ([ISO 29148:2011, page 12](https://sci-hub.do/https://ieeexplore.ieee.org/document/6146379))

I agree with @henningfemmer to the effect that right here there is no risk of ambiguity, because personally I do not link _instructions_ to _having functions_. And so I'm also joining @Peldan in saying, that the pronoun would best suit here. 

Nevertheless I do not think that the issue holds any significant importance for the project. I just wanted to state my opinion 😅